### PR TITLE
Add a generic form template for feedback submission form

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

File: templates/feedback/submit_feedback.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    {% if field_obj %}
+                      </a>
+                    {% endif %}
+                {% endwith %}
+              {% endfor %}
+            </ol>
+          </div>
+        {% endif %}
+
+        <form action="{{ app_path }}" method="post" id="login-form">
```
Comment (SteveMarshall) [inline templates/feedback/submit_feedback.html]: `id="login-form"`?

**Comment by SteveMarshall** [comment]
Does this depend on ministryofjustice/django-zendesk-tickets#1?

**Comment by ibrechin** [comment]
It doesn't *depend* on it, by which I mean that nothing will break if it is merged before that is. The change to the "Contact Us" link won't work until changes have been made to the frontend apps, but that link was previously not working anyway, so...

**Comment by SteveMarshall** [comment]
@ibrechin Fair enough; I'm happy for it to be merged, then, if @maxf is happy with the markup?

**Comment by maxf** [comment]
I'd rather have each form field visible in the template, as opposed to iterated from the data passed by the controller. As was done in the search form in Cashbook. That way all user visible text is in the template and not in python files in the controller, and styling is more flexible.

**Comment by ibrechin** [comment]
The reason for the more general format is that I considered it likely that the form might have slightly different fields in the different apps. If you would prefer all fields to be specified in the template we probably shouldn't put it in the common package at all, and we will have one template file for each app.

**Comment by maxf** [comment]
Don't we already have one template file for each page?

**Comment by ibrechin** [comment]
Well, the point of this common package is to avoid duplicating templates across the different frontend apps, right? But only where the template would be exactly the same in each app.

**Comment by maxf** [comment]
A generic form template is certainly useful (it should be in its own
partial, in fact). But it does makes front-end styling harder, leading to
not using it for specific pages, eg search. So we can have it in a partial
in the common repo, but we'd still have the issue of user-visible text in
controller code.

**Comment by ibrechin** [comment]
Error messages are defined in the controller though, and they are user visible, so it's never going to be a perfect split.

So what are we talking here? Shall I redo this as an entirely generic include-able form template?

**Comment by maxf** [comment]
Not for this feature, but something to keep in mind when refactoring.

**Pull request merged**